hw5/src/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+const app = express();
+
+var morgan = require('morgan');
+const process = require('process');
+
+const bodyParser = require('body-parser');
+
+const userRouters = require('./routes/user-routes');
+const groupRoutes = require('./routes/groups-routes');
+const userGroup = require('./routes/userGroup-routes');
+
+const { logger } = require('./helper/winstonLogger');
+const { isOperationalError } = require('./helper/errors');
+const port = 3000;
+
+app.use(bodyParser.json());
```

Comment:
you must remove `logger.log` from every controller and put it in a single place (here) as an express middleware